docs/installation.rst

```diff
@@ -99,6 +99,12 @@ Add django-debug-toolbar's URLs to your project's URLconf:
 This example uses the ``__debug__`` prefix, but you can use any prefix that
 doesn't clash with your application's URLs.
 
+> [!NOTE]
```

Comment:
Looking at everything altogether, it's probably better to call out this difference in step 3 "Install the app". This is because we want to let them know earlier on, there's a different way to configure the toolbar.